ripd/ripd.c

```diff
@@ -1290,7 +1290,7 @@ static void rip_response_process(struct rip_packet *packet, int size,
 				memcpy(&ifaddr, ifc->address,
```

Comment:
Looks like you missed:

```c
memcpy(&ifaddr, ifc->address, sizeof(struct prefix_ipv4));
```

Just one line above your change.

I think we should be more careful with this change for `memcpy` because sometimes the source can be smaller than the destination on purpose (example: `struct sockaddr_storage` being copied to a `struct sockaddr_in`, code might know about this but the automated script not). In this case I think `prefix_copy` should be preferable than `memcpy` (or something else I might be missing).

Reply:
Thanks, fixed that one (two :smile:).